code/lib/core-server/src/utils/update-check.ts

```diff
@@ -49,7 +49,7 @@ export function createUpdateMessage(updateInfo: VersionCheck, version: string):
           ${chalk.gray('Upgrade now:')} ${colors.green(upgradeCommand)}
 
           ${chalk.gray('Read full changelog:')} ${chalk.gray.underline(
-            'https://github.com/storybookjs/storybook/blob/next/CHANGELOG.md'
+            `https://github.com/storybookjs/storybook/blob/v${updateInfo.data.latest.version}/CHANGELOG.md`
```

Comment:
This is a great idea! I think it would be sufficient to use the `main` branch, which is always associated with the latest version:

```suggestion
            `https://github.com/storybookjs/storybook/blob/main/CHANGELOG.md`
```